### DISCUSSION TITLE
Trying to figure out how to make weapons.

### DISCUSSION THREAD

**Discussion by nameme1moretime** [nosoop/SM-TFCustomWeaponsX#53] [Q&A]
What am i doing wrong here

[items.txt](https://github.com/nosoop/SM-TFCustomWeaponsX/files/10610451/items.txt)

**Comment by nosoop** [comment]
The error message CWX reports when trying to load that config tells you the following:

> [cwx/cwx.smx] Item uid '{6b614c58-a527-11ed-b9df-0242ac120003}' inherits from unknown item 'TF_WEAPON_SHOTGUN'

`TF_WEAPON_SHOTGUN` is not a valid item &mdash; since you want it to be a Pyro-only weapon you can use `TF_WEAPON_SHOTGUN_PYRO` and drop the `item_class` key and `used_by_classes` section.

You can also use `Upgradeable TF_WEAPON_SHOTGUN_PRIMARY`, but that's more useful for multi-class Shotguns.  Still, you can use that with `used_by_classes` to restrict it to Pyro.
